### PR TITLE
app-editors/atom: Version bump to 1.11.2 (#597442)

### PR DESCRIPTION
**Pull request by elprans**
Gentoo-Bug: https://bugs.gentoo.org/show_bug.cgi?id=597442
Gentoo-Bug: https://bugs.gentoo.org/show_bug.cgi?id=596466

Package-Manager: portage-2.3.0

@gentoo/proxy-maint

### CODE REVIEW

**Comment by gktrk** [comment]
@gentoo/proxy-maint

File: app-editors/atom/atom-1.11.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,398 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit flag-o-matic python-any-r1 eutils multiprocessing rpm
```
Comment (anthonyryan1) [inline app-editors/atom/atom-1.11.2.ebuild]: Are you sure you need eutils here?

Comment (elprans) [inline app-editors/atom/atom-1.11.2.ebuild]: It's necessary for epatch, isn't it?

Comment (elprans) [inline app-editors/atom/atom-1.11.2.ebuild]: Ah, I see.  Didn't realise eapply was part of the EAPI now.  Thanks for pointing this out. I've tested and pushed the fix.

File: app-editors/atom/atom-1.11.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+		if [ -n "${cruft}" ]; then
+			rm ${cruft} || die
+		fi
+	done
+}
+
+_fix_executables() {
+	local _dir="${1}" _node_sb="#!$(get_electron_dir)"/node
+
+	(find -L "${ED}/${_dir}" -maxdepth 1 -mindepth 1 -type f -print || die) \
```
Comment (gktrk) [inline app-editors/atom/atom-1.11.2.ebuild]: why do we require a subshell here? If it's required, why do we not use it in line 325?

File: app-editors/atom/atom-1.11.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+"
+
+RESTRICT="mirror"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="
+	!dev-util/apm
```
Comment (elprans) [inline app-editors/atom/atom-1.11.2.ebuild]: The atom overlay used to have it, can safely drop now.

**Comment by elprans** [comment]
Please see https://github.com/gentoo/gentoo/pull/2836 for a newer version bump.

**Pull request closed without merging**